src/Illuminate/Translation/Translator.php

```diff
@@ -99,12 +99,12 @@ public function has($key, $locale = null, $fallback = true)
      * Get the translation for the given key.
      *
      * @param  string  $key
-     * @param  array  $replace
+     * @param  mixed  $replace
      * @param  string|null  $locale
      * @param  bool  $fallback
      * @return string|array
      */
-    public function get($key, array $replace = [], $locale = null, $fallback = true)
+    public function get($key, mixed $replace = null, $locale = null, $fallback = true)
```

Comment:
This is a breaking change to the method signature.

Reply:
> This is a breaking change to the method signature.

Hey! What do you propose?

Reply:
Best to send this to master

Reply:
> Best to send this to master

What will be the difference to send it to the master?

Reply:
New major versions allow for breaking changes. But I'm not sure we should do these changes. Up to you to try to send it to master or not.